5.5/root/usr/share/container-scripts/mysql/lib.sh

```diff
@@ -0,0 +1,3 @@
+function info-log {
```

Comment:
I'm not much a fan of dashes in function name, something like `log_info` (words deliberately swapped) would create space for potential `log_warning`, `log_error` in the future.